src/superphot_plus/example_benchmarks.py

```diff
@@ -0,0 +1,14 @@
+"""An example module containing simplistic methods under benchmarking."""
```

Comment:
This should probably be moved to benchmarks folder or excluded altogether

Reply:
You're right, I didn't realize I had left it behind. It is gone now!